src/components/RandomBeer.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import Header from './Header';
+
+export default class RandomBeer extends Component {
+  state = {
+    beer: {}
+  }
+
+  componentDidMount() {
+    console.log('ShowBeer.js - component did mount');
+    fetch('https://ih-beers-api2.herokuapp.com/beers/random')
+      .then(res => res.json())
+      .then(result => {
+        console.log(result)
+        this.setState({ beer: result })
+      })
+      .catch(e => console.log(e))
+    }
+  render() {
+      console.log(this.state.beer)
```

Comment:
Just want to mention that  javascript object destructuring is used often to make your code in React more readable. Here, you can write `const {name, image_url, tagline, first_brewed, attenuation_level, description, contributed_by} = this.state.beers` and use it on line 26-32 (you don't have to but it happens a lot in React)